applications/bluesky.md

```diff
@@ -0,0 +1,227 @@
+	- Ecosystem: Solidity has a more mature ecosystem than ink!, with more libraries, frameworks, and tools available. Building a robust ecosystem around ink! would make it more attractive to developers who are looking for a wider range of options when building their applications, starting from increased confidence level via specific examples showcaseing technical advantages backed up with tangible metrics.
+	- Publish blog post in Medium or other media, reaching out to both technical and non-technical audience to better understand ink! smart contract especially against incumbents such as Solidity with not well publicized benefits backed up with concrete feature comparisons and benchmark results.     
+
+
+- What your project is _not_ or will _not_ provide or implement
+  - This is a place for you to manage expectations and to clarify any limitations that might not be obvious
+  - This showcase project is not about developing tooling, performance benchmarks, indusrty specific macros and templates, etc. It simply serves as an example to showcase some of ink! smart contract benefits and potentials. 
+
+
+Things that shouldn’t be part of the application:
+- The (future) tokenomics of your project 
+- For non-infrastructure projects—deployment and hosting costs, maintenance or audits
+- Business-oriented activities (marketing, business planning), events or outreach
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Name of team leader: Daniel Deng
+- Names of team members: 
+	- Daniel Deng, project manager, product and technology leadership at bigtechs and startups, Libra / Diem / Novi / Web3 / smart contract infrastructure 
+	- Bernhard Schuster, former Parity team lead, lecturer at Polkadot Academy, running local Rust meetups, mentor
+	- Andrew Berger, Rust / Substrate develdoper, lecturer of Polkadot Academy, mentor
+
+### Contact
+
+- **Contact Name:** Daniel Deng
+- **Contact Email:** gotellme@proton.me 
+- **Website:** 
+
+### Legal Structure
+
+- **Registered Address:** Address of your registered legal entity, if available. Please keep it in a single line. (e.g. High Street 1, London LK1 234, UK)
+- **Registered Legal Entity:** Name of your registered legal entity, if available. (e.g. Duo Ltd.)
+
+### Team's experience
+
+Lecturers of 2022 Polkadot Academy with in-depth domain knowledge and many years of Substrate / Rust software experiences actively developing in Polkadot ecosystem, running local Rust meetups
+
+### Team Code Repos
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/coax1d
+- https://github.com/drahnr
```

Comment:
I am not involved in the development of this or have any affiliation with `bluesky` CC @HCastano